### DISCUSSION TITLE
Using chia 2.0.0 invalid gigahorse plots

### DISCUSSION THREAD

**Discussion by scoville07** [Chia-Network/chia-blockchain#16160] [Support]
Hello,

I have about 2PB of gigahorse madmax plots and successfully farming with gigahorse farmer

After installing chia 2.0.0 I found chia does maybe not detect my graphics card? I can enable "compressed plots", restart, but the option "enable gpu harvest" is just missing. Also when I create a plot in the GUI the option "bladebit cuda" in the dropdown is not there. And the GUI detects my plots as invalid. My graphics card is a RTX3060 with 12gb ram. The computer has only 64GB RAM, but it is just the farmer.

Regards
Christian

**Comment by majekqwert** [comment]
Gigahorse plots are not compatible with the official ones. You have to replot. And for replot you need min. 256gb ram (for now)

**Reply by scoville07** [reply]
Oh I see, thanks for confirming, I thought just up until chia supports compressed plot.

Do you also know by chance why my chia gui does not show the option "Enable GPU harvesting" and why in the dropdown menu for the available plotters "Bladebit CUDA" is simply missing? Everything is up to date and the farmer is using gigahorse with gpu just fine. It is a windows 10 machine, 64GB RAM, RTX3060 12GB RAM and latest CUDA and Nvidia driver installed.

**Reply by BrandtH22** [reply]
You likely need to have a new config file created as 1.8.1 did not have the parameters needed for compressed harvesting:

1. Stop chia
2. Rename the config or move it to a different directly
3. In CLI run `chia init`
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia

Side note - we can generally provide more thorough and timely support on our discord server: https://discord.gg/chia)
